android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.2.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.3-bin.zip
```

Comment:
Do we really need to switch to fresh major versions immediately after their release?
Maybe consider a more conservative approach and wait for at least one maintenance release being out.

(we've been hit by bugs in fresh updates already, the one that comes to my mind was related to a new agp version and r8 shrinker)

Reply:
It doesn't hurt, but in general, I agree with you.

Reply:
There is also an opposite issue I see often with Android: a late upgrade consumes too much time for the team to properly support it. So there should be a balance.

Reply:
What R8 bug do you have in mind btw? When was it?

Reply:
https://github.com/organicmaps/organicmaps/pull/2718

Reply:
@pastk it's another case. We didn't have a choice, Google forced everyone to use R8.